MSAL/src/native_auth/network/errors/sign_in/MSALNativeAuthSignInChallengeResponseError.swift

```diff
@@ -26,7 +26,7 @@ import Foundation
 
 struct MSALNativeAuthSignInChallengeResponseError: MSALNativeAuthResponseError {
 
-    let error: MSALNativeAuthSignInChallengeOauth2ErrorCode
+    let error: MSALNativeAuthSignInChallengeOauth2ErrorCode?
     let errorDescription: String?
     let errorCodes: [Int]?
     let errorURI: String?
```

Comment:
Can you also check the `MSALNativeAuthInnerError` class? I noticed that the error field isn't optional for that class